lib/liquid/template.rb

```diff
@@ -212,6 +213,12 @@ def render_to_output_buffer(context, output)
 
     private
 
+    BUG_1346_REGEX = /#{TagStart}#{WhitespaceControl}?\s*\w+\s*#{TagStart}/
+
+    def instrument_for_bug_1346(source)
+      Usage.increment("multiple_open_tags") if BUG_1346_REGEX.match?(source)
```

Comment:
This seems too specific to the specific example.  More generally, we want `end_tag_params` to be empty on the `if end_tag_name == block_delimiter` branch in Liquid::Block.parse_body (https://github.com/Shopify/liquid/blob/c7c21e88f018af6a02095cd8161d065d2967ea21/lib/liquid/block.rb#L80) since any end tag markup there would get ignored, whether that is `{% f` or any other non-whitespace characters.